app/models/extended_metadata_type.rb

```diff
@@ -34,6 +34,14 @@ def extended_type?
     supported_type == 'ExtendedMetadata'
   end
 
+  def linked_by_metadata_attributes?
+    ExtendedMetadataAttribute.pluck(:linked_extended_metadata_type_id).compact.include?(id)
```

Comment:
can this not just be `linked_metadata_attributes.any?` and avoid looping over all the attribute ids